src/browser/modules/Stream/Auth/ConnectForm.tsx

```diff
@@ -140,7 +140,9 @@ export default function ConnectForm(props: ConnectFormProps): JSX.Element {
           <H3>Single sign-on</H3>
           {SSOProviders.map((provider: any) => (
             <StyledSSOButtonContainer key={provider.id}>
-              <FormButton onClick={() => authRequestForSSO(provider.id)}>
+              <FormButton
+                onClick={() => authRequestForSSO(provider.id).catch(authLog)}
```

Comment:
No display of an error message on catch?

Reply:
Good _catch_